components/index.ts

```diff
@@ -127,7 +127,10 @@ app.use("/_wf", webFeaturesRouter)
 
 app.get("/", (req: Request, res) => {
     if (PEACOCK_DEV) {
-        res.send("dev active, you need to access the UI from port 3000")
+        res.contentType("text/html")
+        res.send(
+            '<html> PEACOCK_DEV active, please run "yarn webui start" to start the web UI on port 3000 and access it there. </html>',
```

Comment:
Why is this now wrapped with an HTML tag?

Reply:
If I remove the `res.contentType("text/html")` line, I get `TypeError: invalid media type at ServerResponse.send`. The `<html>` flags can be deleted alright. I just wanted to highlight that html is what being sent.